internal/app/singularity/remote_add_test.go

```diff
@@ -0,0 +1,300 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package singularity
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/sylabs/singularity/internal/pkg/remote"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	invalidCfgFilePath = "/not/a/real/file"
+	invalidRemoteName  = "notacorrectremotename"
+	invalidURI         = "really//not/a/URI"
+)
+
+func createInvalidCfgFile(t *testing.T) string {
+	f, err := ioutil.TempFile("", "")
+	if err != nil {
+		t.Fatalf("cannot create temporary configuration file for testing: %s\n", err)
+	}
+
+	path := f.Name()
+
+	// Set an invalid configuration
+	type aDummyStruct struct {
+		NoneSenseRemote string
+	}
+	cfg := aDummyStruct{
+		NoneSenseRemote: "toto",
+	}
+
+	yaml, err := yaml.Marshal(cfg)
+	if err != nil {
+		f.Close()
+		os.Remove(path)
+		t.Fatalf("cannot marshal YANK: %s\n", err)
```

Comment:
Typo: `YANK`->`YAML`

Reply:
fixed

Reply:
Looks like the type is still there

Reply:
Very sorry about that. I am getting tired and tried to go too fast. It is now fixed.